source/tutorials/index.rst

```diff
@@ -3,9 +3,56 @@
 Tutorials
 #########
 
-Explore our tutorials to discover what you can do with |CL|!
+Explore our tutorials to discover what you can do with |CL|! Now conveniently
+sorted by experience level!
+
+.. container:: multicolumns
+
+   .. container:: column narrow
+
+      .. rst-class:: colh3
+
+      Low
+
+      -  :ref:`docker`
+      -  :ref:`flatpak-tutorial`
+      -  :ref:`kata`
+      -  :ref:`proxy`
+      -  :ref:`redis`
+      -  :ref:`smb-desktop`
+      -  :ref:`clear-samba-share-to-Windows`
+      -  :ref:`yubikey-u2f`
+
+   .. container:: column narrow
+
+      .. rst-class:: colh3
+
+      Moderate
+
+      -  :ref:`hadoop`
+      -  :ref:`spark`
+      -  :ref:`broadcom`
+      -  :ref:`fmv`
+      -  :ref:`kubernetes`
+      -  :ref:`kubernetes-bp`
```

Comment:
Will you please move `kubernetes` to High rating? Thanks. 
Docs team agreed on this in our meeting.

Reply:
After this tutorial is moved to High, I'm ready to merge.